src/utils/IdealGraphVisualizer/HierarchicalLayout/src/main/java/com/sun/hotspot/igv/hierarchicallayout/ClusterInputSlotNode.java

```diff
@@ -107,15 +96,12 @@ public Port getInputSlot() {
         return inputSlot;
     }
 
-    public InterClusterConnection getInterBlockConnection() {
-        return interBlockConnection;
-    }
-
     public Port getOutputSlot() {
         return outputSlot;
     }
 
     public Dimension getSize() {
+        int SIZE = 0;
         return new Dimension(SIZE, SIZE);
```

Comment:
You could directly inline `SIZE`:
```suggestion
        return new Dimension(0, 0);
```